extension-style/src/main/java/com/mapbox/maps/extension/style/sources/generated/GeoJsonSource.kt

```diff
@@ -21,7 +25,37 @@ import com.mapbox.maps.extension.style.utils.toValue
  * @see [The online documentation](https://docs.mapbox.com/mapbox-gl-js/style-spec/sources/#geojson)
  *
  */
-class GeoJsonSource(builder: Builder) : Source(builder.sourceId) {
+class GeoJsonSource(
+  builder: Builder
+) : Source(builder.sourceId) {
+
+  var geoJsonParsed = false
```

Comment:
could this be internal? users could always use addOnGeoJsonParsedListener to get the parsed geojson source, if the source is already parsed, we can emit the parsed geojsonsource immediately.

Reply:
hmmm, perhaps I see your point, let me try it out.

Reply:
Applied, thanks!